src/abstractitensornetwork.jl

```diff
@@ -591,20 +591,22 @@ function neighbor_vertices(ψ::AbstractITensorNetwork, T::ITensor)
   return first.(v⃗)
 end
 
-function linkinds_combiners(tn::AbstractITensorNetwork)
+function linkinds_combiners(tn::AbstractITensorNetwork; edges_to_combine=edges(tn))
   combiners = DataGraph(directed_graph(underlying_graph(tn)), ITensor, ITensor)
   for e in edges(tn)
-    C = combiner(linkinds(tn, e); tags=edge_tag(e))
-    combiners[e] = C
-    combiners[reverse(e)] = dag(C)
+    if e ∈ edges_to_combine
+      C = combiner(linkinds(tn, e); tags=edge_tag(e))
+      combiners[e] = C
+      combiners[reverse(e)] = dag(C)
+    end
   end
   return combiners
 end
 
 function combine_linkinds(tn::AbstractITensorNetwork, combiners=linkinds_combiners(tn))
```

Comment:
One way to organize this would be to define two versions:
```julia
function combine_linkinds(tn::AbstractITensorNetwork, combiners::DataGraph)
  # Implementation of combining the link indices
end

function combine_linkinds(tn::AbstractITensorNetwork; edges::Vector{<:Union{Pair,AbstractEdge}}=edges(tn))
  combiners = linkinds_combiners(tn; edges)
  return combine_linkinds(tn, combiners)
end
```

Reply:
Okay yeah this resolves my above concern

Reply:
Although this is technically not allowed surely because `combine_linkinds(tn)' then points to both functions?

Reply:
The version `function combine_linkinds(tn::AbstractITensorNetwork, combiners::DataGraph) [...] end` I proposed above wouldn't provide a default value so they are distinct functions.